### PR TITLE
WinPcap

### PR DESCRIPTION
**Pull request by majkinetor**
I fixed this one, I beleive few tools depend on it.

I couldn't execute AHK using `Start-ChocolateyProcessAsAdmin` so I invoked AHK as is.
If somebody wants to fix it, please do. If not, I will push it as it is.

### CODE REVIEW

**Comment by majkinetor** [comment]
O yeah, it also depends on core extension 1.0.

**Comment by majkinetor** [comment]
I thought its not released yet. Ignore.

**Comment by gep13** [comment]
>@majkinetor said...
I thought its not released yet. Ignore.

Ah, gotcha.  I pushed it last night, and that is why I merged @AdmiringWorm PR for aptana-studio.

File: automatic/WinPcap/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import-module au
+
```
Comment (AdmiringWorm) [inline automatic/WinPcap/update.ps1]: Missing import for the `Get-PackageCacheLocation` function?

**Comment by majkinetor** [comment]
No... i imported core.. no need to cheery pick.

**Comment by AdmiringWorm** [comment]
@majkinetor 
sure, no need to cherry pick. but there isn't any import for core though.

**Comment by majkinetor** [comment]
Meh... lost during branching and merging I guess, it was there :). 

Added.

**Comment by majkinetor** [comment]
If you are not against this line:

```
AutoHotkey $ahkScript $packageArgs.fileFullPath
```

I would merge. I think its what @ferventcoder called antipattern, but I am not 100% sure.

I tried above line as 

```
Start-ChocolateyProcessAsAdmin  "$ahkScript '$($packageArgs.fileFullPath)'" 'Autohotkey'
```

But AHK complained that it can't find the path. I fail to see how can I make it work hence I used direct call.

**Comment by gep13** [comment]
What does $toolsPath contain at runtime? An absolute path, or a relative path?

**Comment by AdmiringWorm** [comment]
@majkinetor I tried a few different ways to call the `Start-ChocolateyProcessAsAdmin`, no luck though.

@gep13 I think the `$toolsPath` is an aboslute path during runtime.

**Comment by majkinetor** [comment]
I almost always use absolute paths. That is the only way to make sure script run no matter what is current dir.

I think we should merge this. If the need arises we can always later fix it.

**Comment by AdmiringWorm** [comment]
I agree.

@gep13 ?

**Pull request merged**